deltacat/io/object_store.py

```diff
@@ -43,6 +43,19 @@ def get_many(self, refs: List[Any], *args, **kwargs) -> List[object]:
         or may not return ordered results.
         """
 
+    def delete(self, obj: object, *args, **kwargs) -> bool:
+        """
+        Delete a single object from the object store.
+        """
+        return self.delete_many([obj])
+
+    def delete_many(self, objects: List[object], *args, **kwargs) -> bool:
```

Comment:
can we take in str of refs? The objects would indicate that we are taking in values instead of keys.

Reply:
Makes sense, I'll change it to `refs: List[Any]` to match the input to `get_many`, since the inputs are the same.